addons/flashdrive/stringtable.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Project name="AE3">
+    <Package name="FlashDrive">
+        <Container name="Interaction">
+            <Key ID="STR_AE3_Flashdrive_Interaction_Connect">
+                <Original>Connect Flash Drive</Original>
```

Comment:
Genrally speaking, please add all currently supported languages to the stringtable and fill in the english version. This is easier for translators.

Reply:
English is filled in?
How am I supposed to translate into languages, I don't know?

Reply:
What I am trying to say is: Add the tags for other languages likes russian, but fill in the english texts into the russian tags, for example. So they are prefilled and can easily be translated.

Reply:
Translators often make errors fill in the tags themselves and create errors when they do so. Finding the errors is very time consuming. I already had that case.